package.js

```diff
@@ -8,8 +8,8 @@ Package.describe({
 Npm.depends({
   // In order to support IE9, we had to fork pagejs and apply
   // this PR: https://github.com/visionmedia/page.js/pull/288
-  'page':'https://github.com/kadirahq/page.js/archive/f29d4d4491178b285b9058c32d74975a4f945dea.tar.gz',
-  'qs':'3.1.0'
+  'page':'https://github.com/adambharris/page.js/archive/3f404dd90ee6d806af046e41b2faa195d940e629.tar.gz',
```

Comment:
Could you send me a PR to our project at `kadirahq/page.js` with the changes. I'd like to merge it.
Then we can use it.